src/udp_engine.cpp

```diff
@@ -199,6 +199,12 @@ void zmq::udp_engine_t::plug (io_thread_t *io_thread_, session_base_t *session_)
         errno_assert (rc == 0);
 #endif
 
+#ifdef SO_REUSEPORT
+        rc = setsockopt (_fd, SOL_SOCKET, SO_REUSEPORT,
+                         reinterpret_cast<char *> (&on), sizeof (on));
+        errno_assert (rc == 0);
```

Comment:
Just in the off chance Windows adds support for this in the future, please add the ifdef ZMQ_HAVE_WINDOWS bit

Reply:
Done. Here's hoping Windows to add this in the future as well.